tests/unit/types/arrays/test_memmap.py

```diff
@@ -173,3 +173,22 @@ def test_convert_dm_to_dam(tmpdir, mocker):
     mock.assert_called()
     assert len(da) == 0
     assert len(dam) == 100
+
```

Comment:
Can you test if manual interruption of the writing process causes corruptions and therefore this error message?